src/components/input/theme.ts

```diff
@@ -1,4 +1,7 @@
+import { DefaultTheme } from '@react-navigation/native';
+
 const theme = {
+  ...DefaultTheme,
```

Comment:
```suggestion
```

I afraid that this line will override theme that I've done in src/styles/theme

Reply:
fixed in [c4e646a](https://github.com/BinaryStudioAcademy/bsa-winter-2022-growup-mobile/pull/16/commits/c4e646a8c21dbfdbaa44c8abd63e2b557e432645)